maven-embedder/pom.xml

```diff
@@ -140,6 +140,12 @@ under the License.
       <artifactId>slf4j-simple</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-jdk14</artifactId>
```

Comment:
Does this introduce JDK 9+ byte code into the classpath? That would be a deal breaker for now.

Reply:
14 = 1.4, code is from 2006 ;)